### DISCUSSION TITLE
Log in page not in light mode

### DISCUSSION THREAD

**Discussion by DovisBomba13** [anthony1x6000/ROBLOX2016stylus#56] [Q&A] [answered]
Hey! how to fix that the log in page isn't in light mode and the header is black instead of blue.
![log in real](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/162736396/f11b655a-443b-4d7b-bda5-bc01658fe5cf)

**Comment by anthony1x6000** [comment, marked as answer]
Roblox just defaults to a dark mode when you're not logged in. There isn't a fix through my userstyle. As this is not really much of an issue, I may force light-mode at login at a later date. 
Edit: Sorry, that is probably not true. The login page likely defaults to whatever theme your browser is set at.
